lib/gcloud/datastore/dataset.rb

```diff
@@ -321,6 +319,133 @@ def transaction
         end
       end
 
+      ##
+      # Create a new Query instance. This is a convenience method to make the
+      # creation of Query objects easier.
+      #
+      # === Parameters
+      #
+      # +kinds+::
+      #   The kind of entities to query. This is optional. (+String+)
```

Comment:
You can query multiple kinds. This is valid:

```ruby
gcloud.run gcloud.query("Users", "Admins)".where("active", "=", true)
```
 But the common usage is to provide a single `kind` value.